firefox-ios/Client/Ecosia/UI/MultiplyImpact/MultiplyImpact.swift

```diff
@@ -15,6 +15,7 @@ class MultiplyImpact: UIViewController, Themeable {
 
     private struct UX {
         private init() {}
+        static let hardcodedDarkTextColor = EcosiaDarkTheme().colors.ecosia.textPrimary
```

Comment:
Intuitively, I'd rather use `textInversePrimary` before realizing it's an entirely different color 👍 😅